### PR TITLE
fix ctc loss first input as logits, not log_probs

### PR DESCRIPTION
**Pull request by zh794390558**
### PR types
Bug fixes

### PR changes
Docs

### Describe
fix ctc loss first input as logits, not log_probs

### CODE REVIEW

**Review by iclementine — APPROVED** [review]
LGTM
